src/Analyzers/CSharp/Tests/UseThrowExpression/UseThrowExpressionTests.cs

```diff
@@ -541,5 +541,18 @@ public A(T? t)
     }
 }");
         }
+
+        [WorkItem(44454, "https://github.com/dotnet/roslyn/issues/44454")]
```

Comment:
📝 Another pull request is already submitted to fix this

Reply:
Awesome. I'll remove this test.